app/components/app/app.js

```diff
@@ -319,21 +318,6 @@ export default class AppComponent extends React.Component {
       );
     }
 
-    if (!this.state.fetchingUser){
-      return this.renderTermsOverlay();
-    }
-
-    return null;
```

Comment:
Minor style nitpick: we usually keep an explicit `return null` when we only return stuff to render on certain `if` conditions. (Or at least that's the style I prefer - it's possible that we're not very consistent in blip... :/)